### DISCUSSION TITLE
Does the newer versions of this repo require more VRAM?

### DISCUSSION THREAD

**Discussion by nanafy** [d8ahazard/sd_dreambooth_extension#591] [Q&A]
Something I've noticed going back between  version c589a359 and the newest repo is that, it seems **that I need more VRAM now.** 

With the newest build, **I have to disable EMA and caching latents** in order to barely squeeze by and get it to train. Whereas before, this was never an issue. I always had both enabled.

What could be the cause of this? Would it be newer diffusion versions or something with the memory attention ( since this option is not present in that build), I have no idea, just curious and trying to understand what's going on.

I am on a 3060 12GB 
Windows 10

**Comment by zahdab** [comment]
try memory attention : xformers

**Reply by nanafy** [reply]
I have it set already :D
Xformers
Fp16
Adam8bit

**Comment by leppie** [comment]
There was a small mem leak that should be fixed now.
